scripts/drupal-vm/config.yml

```diff
@@ -88,7 +88,7 @@ php_packages_extra:
   - imagemagick
 
 # XDebug configuration.
-php_xdebug_version: 2.5.0
+php_xdebug_version: 2.6.1
```

Comment:
This update made the debugging tests work more reliably (see: https://github.com/acquia/blt/pull/3198)